### PR TITLE
PR for DEVOPS-149: Backport poppler libraries to trusty #141

### PR DESCRIPTION
**Pull request by gilleyj**
RE: https://github.com/easybib/ops/issues/141

Backporting poppler to v0.40.0 to trusty

### CODE REVIEW

**Comment by gilleyj** [comment]
Is there anyway I can make all that wrestling with PackageCloud's cookbook go away in my history?

**Comment by gilleyj** [comment]
good as I know to do.

File: php/recipes/module-poppler-pdf.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
-include_recipe 'php::dependencies-ppa'
+apt_repository 'poppler' do
+  uri          node['poppler']['package_uri']
+  arch         node['poppler']['package_arch']
```
Comment (till) [inline php/recipes/module-poppler-pdf.rb]: Look for `is_amd?` usage in the cookbooks and try not to hard-code the arch. Or is it only available for `amd64`?

File: php/attributes/default.rb [hunk 2]
```diff
@@ -59,3 +59,11 @@
 default['amazon-elasticache-cluster-client'] = {
   'php_version' => '7.0'
 }
+
+default['poppler'] = {}
+default['poppler']['package_uri'] = 'https://packagecloud.io/till/poppler-backport/ubuntu/'
+default['poppler']['package_arch'] = 'amd64'
+default['poppler']['package_distro'] = 'trusty'
+default['poppler']['package_components'] = ['main']
+default['poppler']['package_key_uri'] = 'https://packagecloud.io/till/poppler-backport/gpgkey'
```
Comment (till) [inline php/attributes/default.rb]: Can you import the key into cookbook files? See how I did it with amazon-elasticache-client.

**Pull request closed without merging**